src/pc/common/CommonChunkColumn.js

```diff
@@ -53,7 +53,6 @@ class CommonChunkColumn {
   }
 
   setBlockEntity (pos, tag) {
-    // Note: `pos` is relative to the chunk, not the world, tag's XYZ is
```

Comment:
Why the removal of this comment? The blockEntities object is supposed to indeed hold entities relative to the current chunk. The chunk is not necessarily bounded to any world hence it's not supposed to hold global coordinates.